src/common/parallel_for.cpp

```diff
@@ -28,9 +26,9 @@ void ParallelFor::par_for_inc(const bitCapInt begin, const bitCapInt end, Increm
 {
     std::atomic<bitCapInt> idx;
     idx = begin;
-    bitCapInt pStride = ParStride;
+    bitCapInt pStride = PSTRIDE;
```

Comment:
Add a comment indicating that this is declared only for the lambda closure; alternatively, change the lambda [] to just include [&] and I think it won’t be necessary.  Other alternate - use PSTRIDE directly inside the lambda, no closure necessary?

Reply:
I think PSTRIDE can be directly used inside without capture, which is fine.